dune_client/client_async.py

```diff
@@ -40,6 +40,44 @@
 from dune_client.query import QueryBase, parse_query_object_or_id
 
 
+class GetResultParams(NamedTuple):
```

Comment:
Ok so I see you have created all these classes, but its unclear where they are being used. Since there is so much dictionary manipulation happening above... Something seems wrong.

Reply:
Should I add a doc string for their usage? I have re-used these classes as parameters for multiple functions.